### PR TITLE
Add a test of basic CSS value serialization roundtripping behaviour.

### PR DESCRIPTION
**Pull request by jdm**

### CODE REVIEW

**Comment by sideshowbarker** [comment]
This is waiting on review of the changes from @jdm in https://github.com/w3c/web-platform-tests/pull/1427/commits/c34297c029d3390bf527dde24f0179d8a2cd0d68

@Ms2ger?

File: cssom/serialize-values.html [hunk 1]
```diff
@@ -0,0 +1,597 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSSOM</title>
+<meta name="author" title="Josh Matthews" href="mailto:josh@joshmatthews.net">
+<link rel="stylesheet" href="/resources/testharness.css">
```
Comment (zcorpan) [inline cssom/serialize-values.html]: Remove this link

File: cssom/serialize-values.html [hunk 2]
```diff
@@ -0,0 +1,597 @@
...
+<body>
+  <div id="log"></div>
+  <div id="parent"></div>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
```
Comment (zcorpan) [inline cssom/serialize-values.html]: I think these should be in `head`?

File: cssom/serialize-values.html [hunk 3]
```diff
@@ -0,0 +1,597 @@
...
+      var elem = document.createElement('div');
+      document.getElementById('parent').appendChild(elem);
+      var test_data = generator();
+      var style = generate_inline_style(property, test_data);
+      if (!style) {
+        return false;
+      }
+      var t = async_test(style.declaration);
+
+      t.cleanup(function() {
```
Comment (zcorpan) [inline cssom/serialize-values.html]: Should be `add_cleanup`

File: cssom/serialize-values.html [hunk 4]
```diff
@@ -0,0 +1,597 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSSOM</title>
```
Comment (zcorpan) [inline cssom/serialize-values.html]: Could the title be a bit more descriptive?

**Review by zcorpan — COMMENTED** [review]
This is an old PR; consider checking the checkbox in the PR to allow edits from maintainers.

**Comment by jdm** [comment]
Checkbox checked.

**Comment by zcorpan** [comment]
> remote: Permission to jdm/web-platform-tests.git denied to zcorpan.

`¯\_(ツ)_/¯`

I guess I can just land my changes and close this PR...

**Comment by zcorpan** [comment]
```diff
diff --git a/cssom/serialize-values.html b/cssom/serialize-values.html
index 310ad80..8f6f3d2 100644
--- a/cssom/serialize-values.html
+++ b/cssom/serialize-values.html
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>CSSOM</title>
+<title>CSSOM serialize values</title>
 <meta name="author" title="Josh Matthews" href="mailto:josh@joshmatthews.net">
-<link rel="stylesheet" href="/resources/testharness.css">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
 <body>
   <div id="log"></div>
   <div id="parent"></div>
-  <script src="/resources/testharness.js"></script>
-  <script src="/resources/testharnessreport.js"></script>
   <script>
     function iterable(values) {
       var i = 0;
@@ -224,7 +223,7 @@
       }
       var t = async_test(style.declaration);
 
-      t.cleanup(function() {
+      t.add_cleanup(function() {
         document.getElementById('parent').removeChild(elem);
       });
 
```

**Comment by zcorpan** [comment]
https://github.com/w3c/web-platform-tests/commit/9fcccf38b6be00f71ffa6bd6e29c5aa1ef25ee8c

Thanks!

**Comment by jdm** [comment]
Thanks for following up!

**Comment by zcorpan** [comment]
Filed issue for Chromium: https://bugs.chromium.org/p/chromium/issues/detail?id=661174

**Pull request closed without merging**